zipline/src/commonMain/kotlin/app/cash/zipline/EventListener.kt

```diff
@@ -38,12 +38,7 @@ abstract class EventListener {
    * @return any object. This value will be passed back to [callEnd] or [callFailed] when the call
    *     is completed. The base function always returns null.
    */
-  open fun callStart(
-    name: String,
-    service: ZiplineService,
-    functionName: String,
-    args: List<Any?>,
-  ): Any? {
+  open fun callStart(call: ZiplineCall): Any? {
```

Comment:
Public API change!

Reply:
Nice, it's a lot easier now to see the similarities between `callStart` and `callEnd`